src/test/kotlin/racingcar/model/CarManagerTest.kt

```diff
@@ -0,0 +1,50 @@
+package racingcar.model
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import racingcar.entity.Car
+import racingcar.entity.Name
+import racingcar.entity.Position
+
+class CarManagerTest {
+    @Test
+    fun `CarManager를 생성 할 때 cars의 size가 1이하면 IllegalArgumentException 예외 발생`() {
+        assertThrows<IllegalArgumentException> {
+            val names = listOf("test1")
+            val positions = listOf(Position(3))
+            val cars = names.mapIndexed { idx, name -> Car(Name(name), positions[idx]) }
+            val sequentialNumberGenerator = SequentialNumberGenerator(listOf())
```

Comment:
```suggestion
     val cars = listOf(Car(Name("test1")))
```
CarManager의 생성부분을 테스트하기 위한 부분인데 불필요한 부분이 많은거 같아요! 
테스트를 위한 세팅외에는 다른 로직들 (mapIndexed 등)은 최소화하는게 
테스트목적에 대한 신뢰성을 높이곤합니다 👍